app/code/community/Ebizmarts/MailChimp/Helper/Data.php

```diff
@@ -2455,7 +2485,7 @@ protected function _makeForCollectionItem($collection, $mailchimpStoreId, $initi
 
         do {
             $collection->setCurPage($currentPage);
-            $collection->load();
+            $this->_loadItemCollection($collection);
```

Comment:
Unnecessary, please remove entire function.

Reply:
This load is necessary to load the next page (filter) in the collection using the same object. If we remove the Load method of that collection, the page iterated would be always the same.